torch/quantization/_quantize_script.py

```diff
@@ -64,6 +64,16 @@ def __setstate__(self, state):
     linear_packed_params = torch.jit.script(torch.nn.quantized.modules.linear.LinearPackedParams())._c
     conv_packed_params = torch.jit.script(ConvPackedParams())._c
 
+def _preprocess_qconfig_dict_helper(module, qconfig_dict, prefix=''):
+    dict_with_names = {}
```

Comment:
can you just use qconfig_dict here? and filter out the types in the end?

Reply:
I'm using qconfig_dict to access the original qconfigs in the parent function (since that take precedence over anything set here). I just thought this is cleaner to read.

Reply:
I feel it's pretty clear to just use qconfig_dict, but remember copy the qconfig_dict before you modify it.